libs/galley-types/src/Galley/Types/Teams/Intra.hs

```diff
@@ -13,19 +13,22 @@ data TeamStatus
     | PendingDelete
     | Deleted
     | Suspended
+    | Pending
```

Comment:
How about using `PendingActive` to give a bit more info similarly to what we do with `PendingDelete` ?